### ISSUE TITLE
default maximum grade for checkmark mod is inconsistent with auto-changed values

### ISSUE THREAD

**Issue by stemiwe** [academic-moodle-cooperation/moodle-mod_checkmark#39]
tested on checkmark 2019012000

creating a new checkmark mod, the default values for "number of examples" is 10, and for "maximum grade" is 100.

however, as soon as "number of examples" is changed in the form, "maximum grade" is auto-calculated to be the same as "numer of examples".

so if a teacher changes "number of examples" to 9, and then changes it back to 10, the maximum grade will be 10 instead of 100.

this is very confusing and should be changed so that default maximum grade = 10.

**Comment by goggo24** [comment]
That's true. The maximum grade is auto-calculated with the number of exam. At the moment the maximum grade has to be an integral multiple of the amount of examples and the grade is per default 100.
Your suggestion is that the maximum grade is 10 in the beginning (default) instead of 100 - did I get it right?

**Comment by stemiwe** [comment]
yes, that would be consistent - then it would stay the same when the number of examples is changed back to 10.

**Comment by jondo** [comment]
@goggo24 Why has this got the 'question' label when your question was already answered?

**Comment by LisaE14** [comment]
We have the same problem with 2021052802 and Moodle 3.11.13.

Since later the maximum grade is calculated automatically, gradepointdefault should not be used at the beginning. Every moodle instance has a different gradepointdefault and that usually does not fit.
